docs/source/release_notes/release-notes.rst

```diff
@@ -55,6 +55,24 @@ develop
            An exception is thrown to the service who made the request; this service has the opportunity to log at a higher level if desired.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/3069>`__)
 
+    *    - |new|
+         - AtlasDB now supports runtime configuration of throttling for stream stores when streams are written block by block in a non-transactional fashion.
+           Previously, such streams would be written using a separate transaction for each block, though in cases where data volume is high this may still cause load on the key-value-service Atlas is using.
+           Please note that if you wish to use this feature, you will need to regenerate your Atlas schemas and suitably inject the stream persistence configuration into your stream stores.
+           However, if you do not intend to use this feature, no action is required, and your stream stores' behaviour will not be changed.
```

Comment:
We should highlight here that clients should be aware that enabling this throttling will make their `storeStream` calls take longer, likely generating longer transactions, affecting sweep (immutable ts being stale) and increasing chances of rollback on read (not sure if we expose the new stream before it finished to be written).

I understand the reasons for this solution (it's quick to implement and gives us what we want), just think that we should have caution when recommending this for our consumers.

Reply:
I've changed the solution to only affect the non-transactional case. With that modification, while writing streams may take longer, I believe we avoid lengthening any transactions (sleep is never called as part of a transaction).